config.yml

```diff
@@ -20,6 +20,9 @@ menu:
       - name: Organization
         url: /organization/
         weight: 10
+      - name: Community
+        url: /community/
+        weight: 15
```

Comment:
I wonder if it better be in the toolbar or a sub-page in the Organization page.
So either `Home -> Community` or `Home -> Orgnization -> Community`.

What do you think is better?

Reply:
Thank you for your swift reply. Considering the fact that all the sample links on the issue have a separate community page that is listed in the toolbar, I think will be a great idea to list it as a separate page. The organization page also has a community. So, that's why I came into such a conclusion. I'm definitely open to suggestions from others.